### ISSUE TITLE
Build failure: unable to find activemq 5.14.0-SNAPSHOT

### ISSUE THREAD

**Issue by cancilla** [IBMStreams/streamsx.messaging#273]
Building the toolkit results in the following failure

```
----------------
     [exec] [INFO] BUILD FAILURE
     [exec] [INFO] ------------------------------------------------------------------------
     [exec] [INFO] Total time: 13.072 s
     [exec] [INFO] Finished at: 2016-10-24T09:37:09-04:00
     [exec] [INFO] Final Memory: 10M/19M
     [exec] [INFO] ------------------------------------------------------------------------
     [exec] [ERROR] Failed to execute goal on project streamsx.messaging: Could not resolve dependencies for project com.ibm.streamsx.messaging:streamsx.messaging:jar:1.0.0: Could not find artifact org.apache.activemq:activemq-client:jar:5.14.0-SNAPSHOT in apache.snapshots (http://repository.apache.org/content/repositories/snapshots/) -> [Help 1]
     [exec] [ERROR] 
     [exec] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
     [exec] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
     [exec] [ERROR] 
     [exec] [ERROR] For more information about the errors and possible solutions, please read the following articles:
     [exec] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

```

**Comment by cancilla** [comment]
Repository https://repository.apache.org/content/repositories/snapshots/org/apache/activemq/activemq-client/ shows that latest SNAPSHOT is 5.15.0-SNAPSHOT

**Comment by Alex-Cook4** [comment]
Thanks @cancilla. This should now be resolved by: #272
